loglist2/loglist2_test.go

```diff
@@ -143,8 +162,9 @@ func TestJSONMarshal(t *testing.T) {
 			if err := json.Indent(&wantIndented, []byte(test.want), jsonPrefix, jsonIndent); err != nil {
 				t.Fatalf("json.Indent(test.want) = %q, want nil", err)
 			}
-			if diff := pretty.Compare(wantIndented.String(), string(got)); diff != "" {
-				t.Errorf("json.Marshal(): diff +got -want\n%s", diff)
+			dmp := diffmatchpatch.New()
```

Comment:
What improvement does this make over `pretty.Compare()`? We're also using `cmp.Diff()` elsewhere too, so I'm mildly reluctant to add yet another diff library unless it's particularly helpful.

Reply:
It's 2-string comparison. So cmp.Diff shows you one long string of the whole JSON in case of mismatch. Not really helpful finding the diff point.

Reply:
Ah yes I've run into this problem too. I think the reason it's calling `json.Indent()` above is to make it easier to read the diff. I guess that's still helpful in conjunction with diffmatchpatch?